### ISSUE TITLE
ControlIQ API Exception

### ISSUE THREAD

**Issue by skirby99** [jwoglom/tconnectsync#30]
I haven't gotten any t:connect data for 24+ hours and I noticed the logs mentioned a ControlIQ API Exception. How do I fix this?
[tconnectsync-check-output.log](https://github.com/jwoglom/tconnectsync/files/7877749/tconnectsync-check-output.log)

**Comment by jwoglom** [comment]
Hi @skirby99,

It looks like what is happening is that you're logged in to your account, but several datasources are not finding any data, although some data DOES exist in some of the APIs. Not sure why that's happening.

Can you log in OK to the tconnect website? It looks like generally in the log you provided there is a lack of data found. Do you use the Android, iOS and/or desktop uploader tool to import your data? 

The oddest thing here is that the ControlIQ summary endpoint is returning a HTTP 404, which it only appears to do when your user ID is wrong. But the same user ID is returned in sections of the other APIs as containing data. I haven't seen that behavior before, which makes me surmise that there's something subtly different about your account or uploader setup than most.

Thanks,
James

**Comment by skirby99** [comment]
I was able to log into tconnect, but it didn't show anything after 1/13. The more I thought about it, that's when we had our initial training and I changed the tconnect password because the trainer wanted to give the password to our endo. I updated the password in Heroku, but I didn't update it in the app on my daughter's phone. I just updated the password on her phone. I'll let you know if that fixes everything.
